libclamav/matcher-byte-comp.c

```diff
@@ -988,15 +988,15 @@ unsigned char *cli_bcomp_normalize_buffer(const unsigned char *buffer, uint32_t
 void cli_bcomp_freemeta(struct cli_matcher *root, struct cli_bcomp_meta *bm)
 {
 
-    int i = 0;
+    uint32_t i = 0;
 
     if (!root || !bm) {
         return;
     }
 
     /* can never have more than 2 */
```

Comment:
We should remove this comment, I guess.  But yeah looks like a simple fix. I'll do some basic testing.